pkg/util/errors/errors.go

```diff
@@ -77,3 +77,17 @@ func IsInvalidError(err error) bool {
 
 	return false
 }
+
+func IsConflictError(err error) bool {
+	if _, ok := err.(gophercloud.ErrDefault409); ok {
+		return true
+	}
+
+	if errCode, ok := err.(gophercloud.ErrUnexpectedResponseCode); ok {
+		if errCode.Actual == http.StatusConflict {
+			return true
+		}
+	}
```

Comment:
What's this about, btw? Worth a code comment?

Reply:
It's copied from above. I could probably do some codegolf to avoid code duplication, let's see.

Reply:
Aw, I'd need to use reflections to pass type to a deduplicated function. Well, it's not Python, I bet it's not worth it.